packages/ui/src/atom/circular-loader-atom/circular-loader-atom.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <field-atom
+    :class="themeCssClass"
+    :style="sizeStyle"
+  >
+    <svg
+      class="circular-loader-atom-back"
+      :style="sizeStyle"
+    >
+      <circle
+        :r="circleAttrs.r"
+        :cx="circleAttrs.cx"
+        :cy="circleAttrs.cy"
+        :stroke-width="lineThickness"
+      />
+    </svg>
+    <svg
+      class="circular-loader-atom-over"
+      :style="{ ...sizeStyle, strokeDasharray }"
+    >
+      <circle
+        :r="circleAttrs.r"
+        :cx="circleAttrs.cx"
+        :cy="circleAttrs.cy"
+        :stroke="circleAttrs.stroke"
+        :stroke-width="lineThickness"
+      />
+    </svg>
+    <slot />
+  </field-atom>
+</template>
+
+<script>
+import ThemeMixin from '@/mixins/ThemeMixin';
+import FieldAtom from '@/atom/field-atom/field-atom';
+
+// For initial state progress must show `action` like something is processing.
+// But if progress have `0` value, circle will be static and '0%' title.
+// Soo default behavior is always show animated circle without label
+// and if progress more than `1`, show progress title
+const MIN_PROGRESS_VALUE = 1;
+const DEFAULT_VIEW_VALUE = 2;
+const MAX_VALUE = 100;
+
+export default {
+  name: 'VProgressCircle',
+
+  props: {
+    progress: {
+      type: [Number, String],
+      default: 0,
+    },
+
+    animated: {
+      type: Boolean,
+      default: true,
+    },
+
+    size: {
+      type: [Number, String],
+      default: 82,
+    },
+
+    stroke: {
+      type: String,
+      default: '#6e32c9',
```

Comment:
maybe instead raw colors we will use class names, and in styles we will use our endpass color variables?

Reply:
default should be empty and stroke can be defined in css classes

Reply:
It is not good, because we should redefine color directly in the `path`, but if we use class, I will be more difficult to redefine color

Reply:
why? we are not using `path` here, we using `circle` and already have selector like
`.circular-loader-atom.theme-default circle {`

Reply:
How you will redefine it outside of component's styles?

Reply:
why we can't use `currentColor` everywhere inside loader? then we will be able to use `skin` property, as in another components of ui kit

Reply:
all you need is
for `circular-loader-atom.vue`:
in props
```
    stroke: {
      type: String,
      default: '',
    },
```

for `circular-loader-atom.theme-default.scss`:
add this
```
.circular-loader-atom.theme-default .circular-loader-atom-over {
  stroke: var(--endpass-ui-color-primary-7);
}
```

and that's all

Reply:
Ok

Reply:
It will not work correctly, because css definition has weight more that inline style